web/components/home/index.sass

```diff
@@ -0,0 +1,15 @@
+.homepage
+  background: $gradient-purple
+
+  &-email-registration
+    > .text-input
+      width: 300px
```

Comment:
this should probably be `width: 100%` and `max-width: 300px`?

Reply:
omg genius